v3/fleet-local/control/chronos@.service

```diff
@@ -6,12 +6,12 @@ Requires=docker.service
 
 [Service]
 # NOTE: chronos doesn't need the `zk://`
-Environment=ZOOKEEPER=localhost:2181
+Environment=ZOOKEEPER_HOST=localhost:2181
 Environment="IMAGE=etcdctl get /images/chronos"
-Environment="zk_username=etcdctl get /zookeeper/ZK_USERNAME"
-Environment="zk_password=etcdctl get /zookeeper/ZK_PASSWORD"
-Environment="principalframework=etcdctl get /frameworkauth/username"
-Environment="zookeeper_endpoint=etcdctl get /environment/ZOOKEEPER_ENDPOINT"
+Environment="ZK_USERNAME=etcdctl get /zookeeper/username"
+Environment="ZK_PASSWORD=etcdctl get /zookeeper/password"
+Environment="ZK_ENDPOINT=etcdctl get /zookeeper/endpoint"
+Environment="MESOS_USERNAME=etcdctl get /frameworkauth/username"
```

Comment:
So, should we put these in `zookeeper/config/values`? To keep with our `config` format